spec/dummy/config/environments/test.rb

```diff
@@ -25,7 +25,7 @@
   # Show full error reports and disable caching.
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = false
-  config.cache_store = :null_store
+  config.cache_store = :memory_store
```

Comment:
the installer will overwrite this, should/can we make this standard or will that mess something up in other projects?

Reply:
@garyfoster we could make it standard, I don't see any reason why not, and if we run into an issue we can back track as needed, @calebtocco what do you think?